WPMedia/fusion-news-theme-blocks
# [PEN-1557] Optional image param compress

## Description
Created a site property (`shouldCompressImageParams`) that allows for users to opt into image param compression.

## Jira Ticket
- [PEN-1557](https://arcpublishing.atlassian.net/browse/PEN-1557)

## Acceptance Criteria
Allow for people to opt into image param compression.

## Test Steps
Check the body of any content source call for a site that has the property set to `true` and check if the image URLs include `cm=t` instead of `:format(jpg):quality(70)` and vice versa for sites where it's set to `false`. 

## Effect Of Changes
### Before
Compression is hard-coded to false.

### After
Users can set a property to opt in to compressing the thumbor params.

## Dependencies or Side Effects
None, other than the changing of the image URLs in content sources. Previously we were hardcoded to opt out of compression.

## Review Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._
please explain why (so that we can fix it whenever it gets refactored).
